WorkloadManagementSystem/Service/WMSUtilities.py

```diff
@@ -70,6 +73,46 @@ def getCREAMPilotOutput( proxy, pilotRef, pilotStamp ):
   result['StdErr'] = error
   return result
 
+def getARCPilotOutput( proxy, pilotRef ):
+  tmp_dir = mkdtemp()
+  myce = pilotRef.split(":")[1].strip("/")
+  gridEnv = getGridEnv()
+  mySite = getCESiteMapping()['Value'][myce]
+  WorkDB = gConfig.getValue('Resources/Sites/LCG/' + mySite + '/CEs/' + myce + '/JobListFile')
```

Comment:
Yes. It is arguable here, but if for a reason or another mySite is 'mySite/', os.path.join saves you :-)